en/docs/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```
@@ -0,0 +1,618 @@
+- **Map Value**: `xmlRecord`
+- **Result**: `xmlResultAsString`
+- **Result Type**: `xml`
+
+> After saving, you should see `xmldata:toXml` in the flow with the result variable name **xmlResultAsString**.
+
+### 2) Add **io:fileWriteXml** to write the XML to disk
+1. With the **Functions** panel open, search for **`fileWriteXml`** (under **Imported Functions → io**).
+2. Click **fileWriteXml** to insert it after **xmldata:toXml**.
+
+Configure the **fileWriteXml** node:
+- **Path**: `outputXML`  _(this is your output file path variable/string.)_
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)